sopel/bot.py

```diff
@@ -324,13 +329,20 @@ def remove_plugin(self, plugin, callables, jobs, shutdowns, urls):
             self.unregister(func)
 
         # remove URL callback handlers
+        for func in urls:
+            regexes = func.url_regex
+            for regex in regexes:
+                if func in self._url_callbacks.get(regex, []):  # can .get ever fail?
```

Comment:
The only reason this line could fail is if someone replace `_url_callbacks[regex]` by something that isn't a list/tuple:

```python console
>>> d = {}
>>> d['a'] = 2  # will fail because int
>>> d['b'] = []  # will work because list
>>> d['c'] = 'some str'  # will fail because iterable BUT wrong type
>>> None in d['a']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'int' is not iterable
>>> None in d['b']
False
>>> None in d['c']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'in <string>' requires string as left operand, not NoneType
```

Reply:
Oh, I mean more "should it ever be possible for `regex` to not be in `self._url_callbacks` unless we've got a bug in some plugin handling code.

Reply:
I see. Well, that's why you can use `.get` with a default value. However, `[]` could totally fail indeed.

Reply:
`regex not in self._url_callbacks` is probably not a case we need to handle better than "don't crash", so resolving